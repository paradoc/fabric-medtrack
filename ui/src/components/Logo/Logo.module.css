.logo {
  display: grid;
  place-items: center;
  row-gap: 4px;
}

.logo > strong {
  font-weight: bold;
  font-size: 48px;
}

.logo > span {
  font-size: 20px;
  font-weight: bold;
  letter-spacing: 1px;
  color: #666;
  text-transform: uppercase;
}