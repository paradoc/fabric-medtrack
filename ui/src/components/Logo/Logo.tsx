import React from 'react'

import styles from './Logo.module.css'

export default function Logo() {
  return (
    <div className={styles.logo}>
      <strong>Mars</strong>
      <span>Drugstore</span>
    </div>
  )
}
