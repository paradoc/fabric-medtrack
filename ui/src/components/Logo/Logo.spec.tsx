import React from 'react'
import { render } from '@testing-library/react'
import Logo from './Logo'

describe('Logo', () => {
  test('should match snapshot', () => {
    const { container } = render(<Logo />)
    expect(container).toMatchSnapshot()
  })
})
