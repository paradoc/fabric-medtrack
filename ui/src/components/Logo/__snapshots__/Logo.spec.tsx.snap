// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Logo should match snapshot 1`] = `
<div>
  <div
    class="logo"
  >
    <strong>
      Mars
    </strong>
    <span>
      Drugstore
    </span>
  </div>
</div>
`;
